srcpkgs/bees/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'bees'
 pkgname=bees
-version=0.7
+version=0.7.2
 revision=1
 archs="~*-musl"
```

Comment:
why no musl? is this something that could be patched/fixed with minimal effort?

Reply:
To quote @Chocimier from #33356:

> musl will fail as upstream uses glibc extensions.

I don't know how easy that would be to fix, but I don't think it's minimal effort

Reply:
ok, that's fine then